Subject: [PATCH] Updated submodule to latest.
---
Index: src/main/resources/config.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/config.yml b/src/main/resources/config.yml
--- a/src/main/resources/config.yml	(revision 1150f0584a68092a92d9046b8b5e86a3d19d143d)
+++ b/src/main/resources/config.yml	(date 1746472322260)
@@ -15,15 +15,12 @@
   blockball:
     aliases:
       - "soccer"
-      - "football"
   blockballscoreboard:
     aliases:
       - "soccerscoreboard"
-      - "footballscoreboard"
   blockballbossbar:
     aliases:
       - "soccerbossbar"
-      - "footballbossbar"
 
 ############################
 # Debug settings
Index: src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt b/src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt
--- a/src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt	(revision 1150f0584a68092a92d9046b8b5e86a3d19d143d)
+++ b/src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt	(date 1746472440435)
@@ -662,6 +662,24 @@
         arena.meta.redTeamMeta.armor = mapItemsToColoredSerializedItems(items, "16711680")
         arena.meta.blueTeamMeta.armor = mapItemsToColoredSerializedItems(items, "255")
         arena.meta.refereeTeamMeta.armor = mapItemsToColoredSerializedItems(items, "16777215")
+        arena.meta.ballMeta.also {
+            it.size = BallSize.SMALL
+            it.item.skinBase64 =
+                "eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvMmE1MmQ1NzlhZmUyZmRmN2I4ZWNmYTc0NmNkMDE2MTUwZDk2YmViNzUwMDliYjI3MzNhZGUxNWQ0ODdjNDJhMSJ9fX0="
+            it.kickPassDelay = 0
+            it.rotating = false
+            it.enabledInteract = false
+            it.hitBoxRelocation = -0.2
+            it.movementModifier.rollingResistance = 0.04
+            it.movementModifier.shotPassYVelocityOverwrite = 0.0
+            it.soundEffects[BallActionType.ONKICK]!!.pitch = 2.0
+        }
+        val hockey = arrayOfNulls<Item?>(36)
+        for (i in 0 until 9) {
+            hockey[i] = Item("minecraft:iron_hoe,292")
+        }
+        arena.meta.redTeamMeta.inventory = hockey.map { e -> itemService.serializeItem(e) }.toTypedArray()
+        arena.meta.blueTeamMeta.inventory = hockey.map { e -> itemService.serializeItem(e) }.toTypedArray()
 
         arenaRepository.save(arena)
         sender.sendPluginMessage(language.gameCreatedMessage, name)
