Subject: [PATCH] Updated submodule to latest.
---
Index: src/main/resources/config.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/config.yml b/src/main/resources/config.yml
--- a/src/main/resources/config.yml	(revision dffec243f3b474302c1e88f579d674be957a8762)
+++ b/src/main/resources/config.yml	(date 1748682465362)
@@ -15,15 +15,12 @@
   blockball:
     aliases:
       - "soccer"
-      - "football"
   blockballscoreboard:
     aliases:
       - "soccerscoreboard"
-      - "footballscoreboard"
   blockballbossbar:
     aliases:
       - "soccerbossbar"
-      - "footballbossbar"
 
 ############################
 # Debug settings
Index: docs/wiki/docs/api.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/docs/wiki/docs/api.md b/docs/wiki/docs/api.md
--- a/docs/wiki/docs/api.md	(revision dffec243f3b474302c1e88f579d674be957a8762)
+++ b/docs/wiki/docs/api.md	(date 1748682465338)
@@ -31,5 +31,3 @@
     }
 }
 ```
-
-BlockBall broadcasts the following [events.](https://github.com/Shynixn/BlockBall/tree/master/src/main/java/com/github/shynixn/blockball/event)
Index: src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt b/src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt
--- a/src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt	(revision dffec243f3b474302c1e88f579d674be957a8762)
+++ b/src/main/java/com/github/shynixn/blockball/impl/commandexecutor/BlockBallCommandExecutor.kt	(date 1748682766385)
@@ -662,6 +662,27 @@
         arena.meta.redTeamMeta.armor = mapItemsToColoredSerializedItems(items, "16711680")
         arena.meta.blueTeamMeta.armor = mapItemsToColoredSerializedItems(items, "255")
         arena.meta.refereeTeamMeta.armor = mapItemsToColoredSerializedItems(items, "16777215")
+        arena.ball.also {
+            it.render.scale = 0.5
+            it.render.offSetY = -0.2
+            it.render.item.skinBase64 =
+                "eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvMmE1MmQ1NzlhZmUyZmRmN2I4ZWNmYTc0NmNkMDE2MTUwZDk2YmViNzUwMDliYjI3MzNhZGUxNWQ0ODdjNDJhMSJ9fX0="
+            it.hitbox.leftClickRightClickDelayTicks = 0
+            it.render.rotating = false
+            it.hitbox.touchEnabled = false
+            it.hitbox.offSetY = -0.2
+            it.hitbox.rightClickEnabled = false
+            it.rollingResistance = 0.04
+            it.shotPassYVelocityOverwrite = 0.0
+            it.soundEffects[BallActionType.ONKICK]!!.pitch = 2.0
+        }
+        val hockey = arrayOfNulls<Item?>(36)
+        for (i in 0 until 9) {
+            hockey[i] = Item("minecraft:iron_hoe,292")
+        }
+        arena.meta.redTeamMeta.inventory = hockey.map { e -> itemService.serializeItem(e) }.toTypedArray()
+        arena.meta.blueTeamMeta.inventory = hockey.map { e -> itemService.serializeItem(e) }.toTypedArray()
+        arena.meta.customizingMeta.ignoreGoalSize = true
 
         arenaRepository.save(arena)
         sender.sendPluginMessage(language.gameCreatedMessage, name)
@@ -1145,7 +1166,6 @@
                         )
                     )
                 }
-
                 if (arena.meta.redTeamMeta.lobbySpawnpoint != null) {
                     highLights.add(
                         AreaHighlight(
Index: docs/wiki/docs/README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/docs/wiki/docs/README.md b/docs/wiki/docs/README.md
--- a/docs/wiki/docs/README.md	(revision dffec243f3b474302c1e88f579d674be957a8762)
+++ b/docs/wiki/docs/README.md	(date 1748682465345)
@@ -2,8 +2,6 @@
 
 BlockBall is a spigot plugin to play soccer/football games in Minecraft.
 
-Are you looking for the 6.x.x wiki? You can access it via [the WayBackMachine](https://web.archive.org/web/20240811125048/https://shynixn.github.io/BlockBall/build/html/index.html).
-
 ## Features
 
 * Adds a new playable soccer/football lobby game or standalone minigame
@@ -12,7 +10,3 @@
 * PlaceHolderApi Support
 * Multiple games per server
 * Player Stats collecting (Patreon Only)
-
-## Installation
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/VPBYYQUaRCs?si=oJf9aM7RWhpXWJ_y" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
